cmd/crates/soroban-test/tests/fixtures/test-wasms/hello_world/src/lib.rs

```diff
@@ -41,6 +41,17 @@ impl Contract {
         env.storage().persistent().set(&COUNTER, &count);
     }
 
+    pub fn inc_tmp(env: Env) {
+        let mut count: u32 = env.storage().temporary().get(&COUNTER).unwrap_or(0); // Panic if the value of COUNTER is not u32.
```

Comment:
I think it would be great if this change to the counter would be to a completely new key ( i.e. "TMP_COUNTER" ), so that we can be sure that we're testing the right thing.